kn/base/templatetags/base.py

```diff
@@ -25,6 +26,15 @@ def email_filter(value):
 def mark_safe_filter(value):
     return mark_safe(value)
 
+@stringfilter
+@register.filter(name='json')
+def json_filter(data):
+    return mark_safe(json.dumps(data)
+                         .replace('&', '\u0026')
+                         .replace('<', '\u003c')
+                         .replace('>', '\u003e')
+                         .replace('/', '\u002f'))
```

Comment:
In Go zit een functie die de karakters `<`, `>`, `&`, U+2028 en U+2029 escapet voor JavaScript in HTML:
http://golang.org/pkg/encoding/json/#HTMLEscape
Ik vertrouw er wel op dat de makers van Go verstand van zaken hebben.

Die laatste twee zijn wel gek, maar is uitgelegd in deze blog post: http://timelessrepo.com/json-isnt-a-javascript-subset. Ik heb lokaal geprobeerd een soort DOS te creëren hiermee, maar het is me niet gelukt. Blijkbaar escapet Django dit bij de API output.
Overigens is een DOS aanval (een syntax error) het enige dat met dit lek mogelijk zou zijn, als Django dit door zou laten. Geen XSS aanval of iets dergelijks.